.github/workflows/build.yaml

```
@@ -0,0 +1,36 @@
+name: ci
+
+on:
+  push:
+
+env:
+  IMAGE_NAME: user/app
+
+jobs:
+  docker:
+    runs-on: self-hosted
+    runs-on: ubuntu-latest
```

actionlint (1.7.12)

[error] key "runs-on" is duplicated in "docker" job. previously defined at line:11,col:5

(syntax-check)